src/jit.rs

```diff
@@ -866,6 +867,11 @@ fn emit_set_exception_kind<E: UserDefinedError>(jit: &mut JitCompiler, err: Ebpf
     X86Instruction::store_immediate(OperandSize::S64, R10, X86IndirectAccess::Offset(8), err_kind as i64).emit(jit)
 }
 
+#[inline]
+fn emit_indirect_branch_target<E: UserDefinedError>(jit: &mut JitCompiler) -> Result<(), EbpfError<E>> {
+    X86Instruction::end_branch().emit(jit)
```

Comment:
Should we put a feature flag here?

Reply:
Not necessary, in fact I would remove the entire `emit_indirect_branch_target` function and only let the `X86Instruction::end_branch()` in.